documents/tinyengine/api/frontend-api/canvas-api.md

```
@@ -0,0 +1,176 @@
+   setDataSourceMap // 转接的方法， 调用render的setDataSourceMap
+  
+   getDataSourceMap// 转接的方法， 调用render的getDataSourceMap
+  
+  }
+  ```
+
+
+
+### renderer对象（来自画布实例）
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)